AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -365,6 +367,15 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         Amplify.Hub.dispatch(to: .dataStore, payload: networkStatusEventPayload)
     }
 
+    func cancelEmitter() {
```

Comment:
nit: you're cancelling multiple emitters right? :) 
```suggestion
    func cancelEmitters() {
```

Reply:
Updated